### PR TITLE
FAQ - update 'close_app' entry for #971

### PR DESCRIPTION
**Pull request by heinezen**
Changelog

- Change german heading to match english heading
    - Change answer from "Nein" to "Ja" accordingly
- Add hint for Android location services in english version
- Add hint that app does not have to be open to german version
- Mention EXposure Notification system
- Change recommend check time to once a day (without mentioning 24 hour interval)
- Update german link texts to match headings of FAQ entries

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -535,8 +535,8 @@
...
                         "active": true,
                         "textblock": [
-                            "Yes. As long as you have activated exposure logging in the app and Bluetooth is on, encounters are logged in the background. The app itself doesn't have to be open for this. You can minimize it or use 'swipe-to-quit' to close it.",
-                            "To be on the safe side, open the app once a day after 24 hours. You'll find more info at <a href='https://www.coronawarn.app/en/#howto' target='_blank' rel='noopener noreferrer'>This is how the app works best</a>.",
+                            "Yes. As long as you have activated exposure logging in the app and Bluetooth (and Location Services on Android) is on, encounters are logged in the background by the Exposure Notification system integrated in iOS/Android. The app itself doesn't have to be open for this. You can minimize it or use 'swipe-to-quit' to close it.",
+                            "To be on the safe side, open the app once a day. You'll find more info at <a href='https://www.coronawarn.app/en/#howto' target='_blank' rel='noopener noreferrer'>This is how the app works best</a>.",
```
Comment (MikeMcC399) [inline src/data/faq.json:539]: Suggest changing to relative reference
href='https://www.coronawarn.app/en/#howto' =>
href='../#howto'

File: src/data/faq_de.json [hunk 2]
```diff
@@ -532,14 +532,14 @@
...
-                            "Nein. Solange Sie in der App die Risiko-Ermittlung aktiviert und Bluetooth (und bei Android den Standort) eingeschalten haben, werden Begegnungen aufgezeichnet. Sie können die App schließen, sowohl durch Minimieren als auch durch 'Swipe-to-quit'.",
-                            "Sicherheitshalber öffnen Sie die App täglich nach Ablauf von 24 Stunden einmal. Weitere Hinweise finden Sie unter <a href='https://www.coronawarn.app/de/#howto' target='_blank' rel='noopener noreferrer'>So funktioniert die App am besten</a>.",
+                            "Ja. Solange Sie in der App die Risiko-Ermittlung aktiviert und Bluetooth (und bei Android die Standort-Dienste) eingeschalten haben, werden Begegnungen vom Exposure Notfication Framework von iOS/Android aufgezeichnet. Die App selbst muss dafür nicht geöffnet bleiben. Sie können die App schließen, sowohl durch Minimieren als auch durch 'Swipe-to-quit'.",
+                            "Sicherheitshalber können Sie die App einmal täglich öffnen. Weitere Hinweise finden Sie unter <a href='https://www.coronawarn.app/de/#howto' target='_blank' rel='noopener noreferrer'>So funktioniert die App am besten</a>.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:540]: Suggest changing to relative reference
href='https://www.coronawarn.app/de/#howto' =>
href='../#howto'

**Review by MikeMcC399 — COMMENTED** [review]
According to https://www.coronawarn.app/en/faq/#android_location
it is not necessary to have location setting enabled 
for Android 11 and the Corona-Warn-App version 1.5 or above.

My Samsung Galaxy A50 updated itself at the weekend from Android 10 to Android 11, so I imagine there will be more and more devices running Android 11 as time goes on.

File: src/data/faq_de.json [hunk 3]
```diff
@@ -532,14 +532,14 @@
...
+                        "title": "Funktioniert die Aufzeichnung der Begegnungen auch, wenn ich die App schließe?",
                         "anchor": "close_app",
                         "active": true,
                         "textblock": [
-                            "Nein. Solange Sie in der App die Risiko-Ermittlung aktiviert und Bluetooth (und bei Android den Standort) eingeschalten haben, werden Begegnungen aufgezeichnet. Sie können die App schließen, sowohl durch Minimieren als auch durch 'Swipe-to-quit'.",
-                            "Sicherheitshalber öffnen Sie die App täglich nach Ablauf von 24 Stunden einmal. Weitere Hinweise finden Sie unter <a href='https://www.coronawarn.app/de/#howto' target='_blank' rel='noopener noreferrer'>So funktioniert die App am besten</a>.",
+                            "Ja. Solange Sie in der App die Risiko-Ermittlung aktiviert und Bluetooth (und bei Android die Standort-Dienste) eingeschalten haben, werden Begegnungen vom Exposure Notfication Framework von iOS/Android aufgezeichnet. Die App selbst muss dafür nicht geöffnet bleiben. Sie können die App schließen, sowohl durch Minimieren als auch durch 'Swipe-to-quit'.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:539]: Notfication => Notification

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One typo plus suggestions

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json]: good idea!

File: src/data/faq_de.json (hunk 2 above)
Comment (heinezen) [reply src/data/faq_de.json]: fixed

File: src/data/faq_de.json (hunk 3 above)
Comment (heinezen) [reply src/data/faq_de.json]: fixed

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
image

**Comment by dsarkar** [comment]
image

**Pull request merged**